src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,24 @@
+package duke.task;
+
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+        this.DESCRIPTOR = "[T]";
+    }
+
+    @Override
+    public String getDescriptor() {
+        return DESCRIPTOR;
+    }
+
+    @Override
+    public String toString() {
+        return DESCRIPTOR + " | " + (this.isDone ? "1 | " : "0 | ") + this.description;
```

Comment:
You do not need to use the `this` keyword unless it is ambiguous (shadowed by method parameter)